docs/releases/4.2.txt

```diff
@@ -365,6 +365,9 @@ Miscellaneous
 * The undocumented ``negated`` parameter of the
   :class:`~django.db.models.Exists` expression is removed.
 
+* Setting ``update_fields`` in an overwritten ``Model.save()`` method may now
+  be required.
```

Comment:
I was thinking about a separate section in the_"Backwards incompatible changes in 4.2"_ with a short explanation, e.g,
```
Backwards incompatible changes in 4.2
=====================================

...

Setting update_fields in Model.save() may now be required
---------------------------------------------------------

...


Miscellaneous
-------------
```

Reply:
Gave it another go :+1: 
Wasn't sure if I should specifically mention `update_or_create` here? What do you think?

Reply:
We should describe what has changed (with a rationale) and how that may impact the custom ``save()` `implementations. For example:

_"In order to avoid updating unnecessary columns, ``QuerySet.update_or_create()`` now passes ``update_fields`` to the ``Model.save()`` calls. As a consequence, any fields modified in the custom ``save()`` methods should be added to the ``update_fields`` keyword argument before calling ``super()``."_